qt/scientific_interfaces/Inelastic/Analysis/IndirectFitDataPresenter.h

```diff
@@ -25,7 +25,7 @@ class IIndirectDataAnalysisTab;
 
 class MANTIDQT_INELASTIC_DLL IIndirectFitDataPresenter {
 public:
-  virtual void handleAddData(IAddWorkspaceDialog const *dialog) = 0;
+  virtual void handleAddData(MantidQt::MantidWidgets::IAddWorkspaceDialog const *dialog) = 0;
```

Comment:
This is more of a general comment that applies to all namespaces. 
I think in some of these functions is not necessary to add the full namespace (only MantidWidgets::IAddWorkspaceDialog) or even `IAddWorkspaceDialog` might do. 
I think this is better for clearer code, but in other files from the IDA namespace, like `InelasticDataManipulationElwinTab` you call the workspace dialog without using namespace. 

So to keep it more consistent, can we try to add the namespace MantidWidgets::IAddWorkspaceDialog in most places? Or if you have a better solution, let me know.